doc/rados/command/list-inconsistent-obj.json

```diff
@@ -0,0 +1,185 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "epoch": {
+      "description": "Scrub epoch",
+      "type": "integer"
+    },
+    "inconsistents": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "object": {
+            "description": "Identify a Ceph object",
+            "type": "object",
+            "properties": {
+              "name": {
+                "type": "string"
+              },
+              "nspace": {
+                "type": "string"
+              },
+              "locator": {
+                "type": "string"
+              },
+              "version": {
+                "type": "integer",
+                "minimum": 0
+              },
+              "snap": {
+                "oneOf": [
+                  {
+                    "type": "string",
+                    "enum": [ "head", "snapdir" ]
+                  },
+                  {
+                    "type": "integer",
+                    "minimum": 0
+                  }
+                ]
+              }
+            },
+            "required": [
+              "name",
+              "nspace",
+              "locator",
+              "version",
+              "snap"
+            ]
+          },
+          "all_shards": {
+            "description": "Union of all shard errors",
+            "type": "array",
+            "items": {
+              "enum": [
+                "missing",
+                "stat_error",
+                "read_error",
+                "data_digest_mismatch",
+                "omap_digest_mismatch",
+                "size_mismatch",
+                "attr_value_mismatch",
+                "attr_key_mismatch",
+                "data_digest_mismatch_oi",
+                "omap_digest_mismatch_oi",
+                "size_mismatch_oi",
+                "ec_hash_error",
+                "ec_size_error",
+                "missing_object_info_attr",
+                "corrupt_object_info_attr"
+              ]
+            },
+            "minItems": 0,
+            "uniqueItems": true
+          },
+          "errors": {
+            "description": "Errors related to the analysis of this object",
+            "type": "array",
+            "items": {
+              "enum": [
+                "comparing_random_osd",
```

Comment:
Yes, to your second comment.  That's how it works now.   In the test code with only 1 replica so osd.0 x oi x osd.1 y oi x.  osd.0 is selected as the authoritative copy with no shard errors.  Here is one the json output that is checked for in osd-scrub-repair.sh.

    {
      "all_shards": [
        "data_digest_mismatch",
        "size_mismatch",
        "data_digest_mismatch_oi",
        "size_mismatch_oi"
      ],
      "errors": [],
      "object": {
        "locator": "",
        "name": "OBJ1",
        "nspace": "",
        "snap": "head",
        "version": 3
      },
      "shards": [
        {
          "data_digest": "0x2ddbf8f5",
          "errors": [],
          "omap_digest": "0xef54f3bc",
          "osd": 0,
          "selected": true,
          "size": 7
        },
        {
          "data_digest": "0x2d4a11c2",
          "errors": [
            "data_digest_mismatch",
            "size_mismatch",
            "data_digest_mismatch_oi",
            "size_mismatch_oi"
          ],
          "omap_digest": "0xef54f3bc",
          "osd": 1,
          "selected": false,
          "size": 9
        }
      ]
    },